HOSTING.md

```diff
@@ -33,5 +33,52 @@ To deploy on Render, please follow these steps:
 
 Then click "Save Changes", and your application should now be deployed.
 
+### Deploy vectara-answer on Render Using Render Create Service API
 
+Render provides the APIs to manage the services.
+
+Follow these steps to deploy Vectara Answer with the Render API:
+
+1. Create API Key: After logging in go to Account Settings and create the API Key. Checkout the [Render Docs](https://render.com/docs/api) for more information.
+2. Check out the [Render Create Service API](https://api-docs.render.com/reference/create-service) to see the body params.
```

Comment:
Suggest something like: "Check out the [Render Create Service API](https://api-docs.render.com/reference/create-service), and fill in the 'body' parameters", as follows: 
- Choose "web_service" for 'type'
- pick a name for the service
- In "OwnerID" fill in your Render ownerID
- In "Authorization" put in your Render API key in the "Bearer" box
- Fill in the rest of the variables as shown below
..."
Adeel - And then show item 3 with the final actual body, to guide users more specifically to how to fill in the details more concretely.